Userland/Libraries/LibJIT/GDB.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2023, Jesús Lapastora <cyber.gsuscode@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Checked.h>
+#include <AK/OwnPtr.h>
+#include <AK/Span.h>
+#include <LibJIT/GDB.h>
+
+namespace JIT::GDB {
+
+// Declarations from https://sourceware.org/gdb/current/onlinedocs/gdb.html/Declarations.html#Declarations.
+enum class JitActions : u32 {
+    NoAction = 0,
+    RegisterFn = 1,
+    UnregisterFn = 2,
+};
+
+struct JitCodeEntry {
+    JitCodeEntry* next_entry;
+    JitCodeEntry* prev_entry;
+    RawPtr<char const> symfile_addr;
```

Comment:
```suggestion
    char const* symfile_addr;
```
Looks like you missed one?

Reply:
yeah, I forgot about it. Fixed!